instrumentation/newrelic-scala-monix-api/src/main/scala/com/newrelic/monix/api/Util.scala

```diff
@@ -0,0 +1,26 @@
+package com.newrelic.monix.api
+
+import com.newrelic.agent.bridge.AgentBridge
+import com.newrelic.api.agent.NewRelic
+import monix.eval.Task
+
+object Util {
+  def wrapTrace[A](body: Task[A]): Task[A] =
+    Task.delay({
+      AgentBridge.instrumentation.createScalaTxnTracer
+    }).redeemWith(
+      _ => body,
+      tracer => if(tracer == null) {
+        body
+      } else {
+        for {
+          _ <- Task.delay(NewRelic.getAgent.getTransaction)
+          res <- body.onErrorHandleWith(throwable => {
+            tracer.finish(throwable)
+            Task.raiseError(throwable)
+          })
+          _ <- Task.delay(tracer.finish(172, null))
```

Comment:
Want to change 172 to const at top of file just to make clear what the opcode is? I think this was done for CE3